saas/models/saas_operator.py

```diff
@@ -49,6 +51,9 @@ def _create_db(self, template_db, db_name, demo, password=None, lang='en_US'):
                 db.exp_create_database(
                     db_name, demo, lang, user_password=password)
 
+            if test_clear:
+                tools.config['test_enable'] = True
```

Comment:
It may have other value (e.g. in future versions). Restore original value instead
Also, ``test_clear`` is a name that  difficult to understand what does it mean